test/plots/piePlotTests.ts

```diff
@@ -147,6 +141,22 @@ describe("Plots", () => {
         });
         svg.remove();
       });
+
+      it("formatters are used properly", () => {
+        var data = [
+          { value: 5 },
+          { value: 15 }
+        ];
+        var dataset = new Plottable.Dataset(data);
+        piePlot.addDataset(dataset);
+        piePlot.labelFormatter((n: number) => n + "%");
+        piePlot.renderTo(svg);
+        var texts = svg.selectAll("text")[0].map((n: any) => d3.select(n).text());
+        assert.lengthOf(texts, 2, "both labels are drawn");
+        assert.strictEqual(texts[0], "5%", "The formatter was used to format the first label");
+        assert.strictEqual(texts[1], "15%", "The formatter was used to format the second label");
```

Comment:
Not that it's a big deal since these are unit tests, but using percentages here is quite counterintuitive in a pie plot (pie plots are specifically meant to deal with percentages).  Can we use another string as the appendage instead?  Like... "meters" or "m" or really anything else